CONTRIBUTING.md

```diff
@@ -4,4 +4,64 @@ Thank you for your interest in the Langium project! The following is a set of gu
 
 ## Prerequisites
 
-Developing Langium only requires `npm@^7.7.0`. For this you will need `Node.js 10` or greater.
\ No newline at end of file
+Developing Langium only requires `npm@^7.7.0`. For this you will need `Node.js 10` or greater.
```

Comment:
We should check the actual minimum NodeJS version (see #192). I noticed that we use more recent APIs in some places, so NodeJS 10 certainly doesn't work, but that's fine because it's old and no longer supported.

Looking at https://nodejs.org/en/about/releases/ I think we should use version 12 as minimum version.

Reply:
Ok, so minimum version is Maintenance LTS. Are there further requirements apart from workspaces that we have on a specific npm version?

Reply:
No, the workspaces feature is what we need from npm, and regarding NodeJS we need to be careful not to use API that is only available in versions > 12.